<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1524505834741' stop='1524505841877'>
    <name>Purchasing a new bill pay</name>
    <title>Purchasing a new bill pay</title>
    <test-cases>
        <test-case start='1524505836391' status='passed' stop='1524505838405'>
            <name>should go to checkout page</name>
            <title>should go to checkout page</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='d0521bed-bc49-42f7-97dd-798e2bea2a4c-attachment.png' type='image/png' size='856419'/>
            </attachments>
        </test-case>
        <test-case start='1524505838407' status='passed' stop='1524505839609'>
            <name>should fill out the step 0 form and move to step 1 form</name>
            <title>should fill out the step 0 form and move to step 1 form</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a881318b-502b-4c41-8695-ad793f6ad617-attachment.png' type='image/png' size='883701'/>
            </attachments>
        </test-case>
        <test-case start='1524505839609' status='failed' stop='1524505840770'>
            <name>should fill out step 1 form and move to step 2 form</name>
            <title>should fill out step 1 form and move to step 2 form</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='5eeeae83-f22f-4dc9-b85a-bf93a3ef1c4f-attachment.png' type='image/png' size='878435'/>
            </attachments>
            <failure>
                <message>Failed: No element found using locator: By(xpath, //*[@id="frmPersonalDataPass1"]/div/div[8]/div[1]/button)</message>
                <stack-trace>NoSuchElementError: No element found using locator: By(xpath, //*[@id="frmPersonalDataPass1"]/div/div[8]/div[1]/button)
                    at elementArrayFinder.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:814:27)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)
                    at /usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:668:7
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as click] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at UserContext.it (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:39:38)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)
                From: Task: Run it("should fill out step 1 form and move to step 2 form") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.describe (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:29:5)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:4:1)
                    at Module._compile (module.js:660:30)
                    at Object.Module._extensions..js (module.js:671:10)
                    at Module.load (module.js:573:32)
                    at tryModuleLoad (module.js:513:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1524505840772' status='failed' stop='1524505841308'>
            <name>should fill out the step 2 form and move to step 3 form</name>
            <title>should fill out the step 2 form and move to step 3 form</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='ae9e007a-7220-4ec8-931b-c34b766392d9-attachment.png' type='image/png' size='878392'/>
            </attachments>
            <failure>
                <message>Failed: invalid element state: Element is not currently interactable and may not be manipulated
                  (Session info: chrome=66.0.3359.117)
                  (Driver info: chromedriver=2.36.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8),platform=Mac OS X 10.13.3 x86_64)</message>
                <stack-trace>InvalidElementStateError: invalid element state: Element is not currently interactable and may not be manipulated
                  (Session info: chrome=66.0.3359.117)
                  (Driver info: chromedriver=2.36.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8),platform=Mac OS X 10.13.3 x86_64)
                    at Object.checkLegacyResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)
                From: Task: WebElement.clear()
                    at Driver.schedule (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at WebElement.schedule_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
                    at WebElement.clear (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2351:17)
                    at actionFn (/usr/local/lib/node_modules/protractor/built/element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:461:65)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as clear] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as clear] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at UserContext.it (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:44:44)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at UserContext.it (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:44:52)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)
                From: Task: Run it("should fill out the step 2 form and move to step 3 form") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.describe (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:42:5)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:4:1)
                    at Module._compile (module.js:660:30)
                    at Object.Module._extensions..js (module.js:671:10)
                    at Module.load (module.js:573:32)
                    at tryModuleLoad (module.js:513:12)</stack-trace>
            </failure>
        </test-case>
        <test-case start='1524505841309' status='failed' stop='1524505841859'>
            <name>should fill out the step 3 form and show the protocol number informing that the request was sent</name>
            <title>should fill out the step 3 form and show the protocol number informing that the request was sent</title>
            <labels/>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Screenshot' source='a51eb382-f358-4488-b76b-323722f7969c-attachment.png' type='image/png' size='878435'/>
            </attachments>
            <failure>
                <message>Failed: element not visible
                  (Session info: chrome=66.0.3359.117)
                  (Driver info: chromedriver=2.36.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8),platform=Mac OS X 10.13.3 x86_64)</message>
                <stack-trace>ElementNotVisibleError: element not visible
                  (Session info: chrome=66.0.3359.117)
                  (Driver info: chromedriver=2.36.540469 (1881fd7f8641508feb5166b7cae561d87723cfa8),platform=Mac OS X 10.13.3 x86_64)
                    at Object.checkLegacyResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/error.js:546:15)
                    at parseHttpResponse (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:509:13)
                    at doSend.then.response (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/http.js:441:30)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)
                From: Task: WebElement.sendKeys()
                    at Driver.schedule (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:807:17)
                    at WebElement.schedule_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2010:25)
                    at WebElement.sendKeys (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/webdriver.js:2174:19)
                    at actionFn (/usr/local/lib/node_modules/protractor/built/element.js:89:44)
                    at Array.map (&lt;anonymous>)
                    at actionResults.getWebElements.then (/usr/local/lib/node_modules/protractor/built/element.js:461:65)
                    at ManagedPromise.invokeCallback_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:1376:14)
                    at TaskQueue.execute_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3084:14)
                    at TaskQueue.executeNext_ (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:3067:27)
                    at asyncRun (/usr/local/lib/node_modules/protractor/node_modules/selenium-webdriver/lib/promise.js:2927:27)Error
                    at ElementArrayFinder.applyAction_ (/usr/local/lib/node_modules/protractor/built/element.js:459:27)
                    at ElementArrayFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:91:29)
                    at ElementFinder.(anonymous function).args [as sendKeys] (/usr/local/lib/node_modules/protractor/built/element.js:831:22)
                    at UserContext.it (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:53:35)
                    at &lt;anonymous>
                    at process._tickCallback (internal/process/next_tick.js:160:7)
                From: Task: Run it("should fill out the step 3 form and show the protocol number informing that the request was sent") in control flow
                    at UserContext.&lt;anonymous> (/usr/local/lib/node_modules/protractor/node_modules/jasminewd2/index.js:94:19)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at QueueRunner.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4199:10)
                    at Spec.queueRunnerFactory (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:909:35)
                    at Spec.execute (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:526:10)
                    at UserContext.fn (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:5340:37)
                    at attempt (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4297:26)
                    at QueueRunner.run (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4217:20)
                    at runNext (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4257:20)
                From asynchronous test: 
                Error
                    at Suite.describe (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:51:5)
                    at addSpecsToSuite (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1107:25)
                    at Env.describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:1074:7)
                    at describe (/usr/local/lib/node_modules/protractor/node_modules/jasmine-core/lib/jasmine-core/jasmine.js:4399:18)
                    at Object.&lt;anonymous> (/Users/jvitor/Documents/Code/protractor-poc/feature/step_definitions/checkout_spec.js:4:1)
                    at Module._compile (module.js:660:30)
                    at Object.Module._extensions..js (module.js:671:10)
                    at Module.load (module.js:573:32)
                    at tryModuleLoad (module.js:513:12)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>